### PR TITLE
[IE] Refactor Clamp implementation

### PR DESCRIPTION
**Pull request by mingmingtasd**
PTAL, thanks! @fujunwei

### CODE REVIEW

File: src/webnn_native/openvino/ienn/src/ie_model.cpp [hunk 1]
```diff
@@ -270,7 +270,14 @@ ie_operand_t* Model::AddClamp(ie_operand_t* input,
                               ie_clamp_options_t* options) {
   auto input_node = name_node_map_[input->name];
   std::shared_ptr<ngraph::Node> clamp_node;
-  if (options->minDimensionsCount == 0 && options->maxDimensionsCount == 0) {
+  // If minValue and maxValue are both scalars with shape {1} or shape {}, use
+  // native Clamp to create graph. Otherwise, due to the limitation of Clamp's
+  // attributes type, use Maximum and Minimum to implement Clamp as a
+  // workaround. Note that this workaround may cause performance decline in
```
Comment (huningxin) [inline src/webnn_native/openvino/ienn/src/ie_model.cpp:276]: As the [clamp spec ](https://webmachinelearning.github.io/webnn/#api-mlgraphbuilder-clamp) mentions, the clamp implementation "can be generically emulated from the usage of other operations" (min and max). So this approach should be called as "emulation" other than "workaround". Please update the comment, thanks!

**Review by huningxin — APPROVED** [review]
LGTM with a nit. Thanks!

**Comment by huningxin** [comment]
@mingmingtasd , thanks for your PR. Next time, you can request others to review by clicking the "request review" button under "reviewers". That would help notify the reviewers.

**Comment by mingmingtasd** [comment]
> @mingmingtasd , thanks for your PR. Next time, you can request others to review by clicking the "request review" button under "reviewers". That would help notify the reviewers.
Got it, thanks! @huningxin

File: src/webnn_native/openvino/ienn/src/ie_model.cpp (hunk 1 above)
Comment (mingmingtasd) [reply src/webnn_native/openvino/ienn/src/ie_model.cpp]: Thanks! I have updated comments as you suggested, PTAL. @huningxin

**Review by huningxin — APPROVED** [review]

**Pull request merged**